### ISSUE TITLE
Border is getting disappeared upon setting focus on "Add Package" button and "Package source" dropdown.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#364]
### Steps to Reproduce

1. Change the High contrast standards to High Contrast black before opening the application.
2. Launch Xamarin Workbooks application and create a new workbook.
3. Navigate to File -> Add Package. Invoke "Add Package" . It will open "Package Manager" window.
3. Type "Newton" in Search NuGet package edit box to get package list and navigate to package list. Select any package to add.
4.Observe the border of "Package Source" dropdown and "Add Package" button upon navigation on to them.

### Expected Behavior
Border should not disappear.
### Actual Behavior
 Border is getting disappeared upon setting focus on "Add Package" button and "Package source" dropdown.
Note: In High contrast white dotted rectangular focus appears on "Add Package" button and "Package source" dropdowns.
### User Impact
User will be confused with loss of border
### Screenshots
![16 hcb 2](https://user-images.githubusercontent.com/22956130/40725154-5255e908-6440-11e8-8cbd-7c6e3c7c9207.png)
![16 hcb](https://user-images.githubusercontent.com/22956130/40725156-52ab79ea-6440-11e8-825b-9f05a90d267f.png)
![16 hcw 1](https://user-images.githubusercontent.com/22956130/40725158-5339c416-6440-11e8-8391-99809a370aa0.png)
![16 hcw 2](https://user-images.githubusercontent.com/22956130/40725159-536b711e-6440-11e8-9433-240fb4a774f9.png)
![16 normal](https://user-images.githubusercontent.com/22956130/40725161-53d17fcc-6440-11e8-90e2-18c71f22f4d7.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)
